go/datas/remote_database_handlers.go

```diff
@@ -341,7 +341,7 @@ func handleRootGet(w http.ResponseWriter, req *http.Request, ps URLParams, rt ch
 	if req.Method != "GET" {
 		d.Panic("Expected get method.")
 	}
-
+	rt.Rebase()
```

Comment:
yeah, let's not do this. i think right now we should depend on ChunkStores being opened newly for each request.  I think there's some cool work to be done here (making serving ChunkStores efficient), but I think it needs to be designed.